File: lib/src/main/java/ir/arefdev/irdebitcardscanner/OnScanListener.java
package ir.arefdev.irdebitcardscanner;

import android.graphics.Bitmap;

import java.util.List;

interface OnScanListener {

	void onPrediction(final String number, final Expiry expiry, final Bitmap bitmap,
					  final List<DetectedBox> digitBoxes, final DetectedBox expiryBox);

	void onFatalError();

}

File: lib/src/main/java/ir/arefdev/irdebitcardscanner/OnScanListener.kt
package ir.arefdev.irdebitcardscanner

import android.graphics.Bitmap

interface OnScanListener {

    fun onPrediction(
        number: String?,
        expiry: Expiry?,
        bitmap: Bitmap,
        digitBoxes: List<DetectedBox>,
        expiryBox: DetectedBox?
    )

    fun onFatalError()
}
